### ISSUE TITLE
if my client management.port not equal server.port, spring-boot-admin doesn't show Details -> Memory

### ISSUE THREAD

**Issue by treeleafj** [codecentric/spring-boot-admin#246]
my client bootstrap.yml:
```ymal
 #server:
  port: 7000
management:
  port: 7700
......
```
management.port not equal server.port!
spring-boot 1.4.0.RELEASE
spring-boot-admin (1.4.1):
Details->Memory:
![qq 20160810170421](https://cloud.githubusercontent.com/assets/3805088/17548103/f22a6eac-5f1c-11e6-9fb2-17c0f2f210c8.png)

```log
2016-08-10 16:57:47.130  INFO 9568 --- [nio-9091-exec-1] o.s.c.n.zuul.web.ZuulHandlerMapping      : Mapped URL path [/api/applications/97564b0d/heapdump/**] onto handler of type [class de.codecentric.boot.admin.zuul.OptionsDispatchingZuulController]
2016-08-10 16:58:15.567  INFO 9568 --- [freshExecutor-0] d.c.b.a.registry.ApplicationRegistry     : New Application Application [id=38a227db, name=USERSERVICE, managementUrl=http://USER-20160426ME:7000, healthUrl=http://USER-20160426ME:7700/health, serviceUrl=http://USER-20160426ME:7000] registered 
2016-08-10 16:58:16.206  INFO 9568 --- [freshExecutor-0] d.c.b.a.registry.ApplicationRegistry     : New Application Application [id=babdf218, name=ADMIN, managementUrl=http://USER-20160426ME:9091, healthUrl=http://USER-20160426ME:9091/health, serviceUrl=http://USER-20160426ME:9091] registered 
2016-08-10 16:58:16.212  INFO 9568 --- [nio-9091-exec-2] o.s.c.n.zuul.web.ZuulHandlerMapping      : Mapped URL path [/api/applications/38a227db/health/**] onto handler of type [class de.codecentric.boot.admin.zuul.OptionsDispatchingZuulController]
```

**Comment by joshiste** [comment]
Are you using our client or one of the discovery options?

**Comment by treeleafj** [comment]
yes, client using @EnableDiscoveryClient;

**Comment by joshiste** [comment]
From the reference Guide:
> If you want to customize the default conversion of services you can either add health.path, management.port and/or mangament.context-path entries to the services metadata.

http://codecentric.github.io/spring-boot-admin/1.4.1/#_serviceinstanceconverter

This will fix your problem

**Comment by treeleafj** [comment]
I added 
```ymal
eureka:
  instance:
    metadataMap:
      management.port: 7700
```
to client bootstrap.yml is ok;

**Comment by pitinga** [comment]
I have a similar problem. I changed the management port too and the "Environment manager" (image below) disapeared. The eureka metadataMap already has the correct management port. I'm using Spring Boot Admin 1.5.7. Everything else seens to be working fine.

------------

![image](https://user-images.githubusercontent.com/7571851/44115271-d9c33c7c-9fe3-11e8-9e37-eb9de2fc59df.png)

------------

![image](https://user-images.githubusercontent.com/7571851/44115115-7041fa04-9fe3-11e8-89c4-e3bf8ec15b01.png)

------------

![image](https://user-images.githubusercontent.com/7571851/44115230-b49f1100-9fe3-11e8-8c45-c2384dbcf445.png)

**Comment by pitinga** [comment]
waiting-for-feedback

**Comment by pitinga** [comment]
The problem is in Spring Boot Actuator that returns error 500 for "HEAD /actuator/env" instead of error 405 that is what Spring Boot Admin expects. Bugfix will be released on Spring Boot 1.5.16.

https://github.com/spring-projects/spring-boot/issues/14084
